tests/extract/test_triples.py

```diff
@@ -92,9 +92,10 @@ def sss_doc(lang_en):
                 (["She", "friend"], ["sell"], ["sea", "shells"]),
                 (["She", "friend"], ["throw"], ["sea", "shells"]),
             ],
-        ),
+        )
     ],
 )
+
```

Comment:
looks like your code editor is making some spurious changes (here and elsewhere) and that aren't PEP-compliant / black-enforced. we'll want to fix these before any merge.